### PR TITLE
669 thousand separators

### PR DESCRIPTION
**Pull request by raduiordache**
Did I do this right?

### CODE REVIEW

File: tests/models/test_representations.py [hunk 1]
```diff
@@ -41,8 +42,10 @@ def setUp(self):
         self.test_model = product_root.add_model(name=self.test_model_name, multiplicity=Multiplicity.ONE)
         self.prop_model = self.test_model.add_property(
             name=self.test_prop_name,
-            type=self.property_type,
+            property_type=self.property_type,
         )  # type: AnyProperty
+        if self.options:
+            self.prop_model.options = self.options
```
Comment (JelleBoersma) [inline tests/models/test_representations.py]: I would modify the sub-class of test specifically for the single-select property if the options have to be set. Now you're adding a work-around to this superclass instead. Simplify create another setUp() method that calls super().setUp(), but with the options attached.

File: pykechain/models/representations/representations.py [hunk 2]
```diff
@@ -29,6 +29,25 @@ class SignificantDigits(DecimalPlaces):
     rtype = PropertyRepresentation.SIGNIFICANT_DIGITS
 
 
+class ThousandsSeparator(BaseRepresentation):
+    """Representation for integer or floating-point value properties."""
+
+    rtype = PropertyRepresentation.THOUSANDS_SEPARATOR
+
```
Comment (JelleBoersma) [inline pykechain/models/representations/representations.py]: There is no _config_value_key required for this representation type? (this is taken from the back-end JSON)

Comment (raduiordache) [reply pykechain/models/representations/representations.py]: No _config_value_key for this representation type indeed

File: tests/models/test_representations.py (hunk 1 above)
Comment (raduiordache) [reply tests/models/test_representations.py]: You're right, I fixed it now

**Pull request merged**